database/migrations/2020_05_11_100000_create_participants_table.php

```diff
@@ -12,7 +12,7 @@ public function up()
     {
         Schema::create('participants', function (Blueprint $table) {
             $table->id();
-            $table->morphs('user')->nullable();
+//            $table->morphs('user');
```

Comment:
This comes close to passing model tests, but since morphs create both an `id` and `type` column, the support user model for testing does not work in the scenario.

Reply:
If this is a blocker - we can restore it to user_id.

Reply:
I prefer a nullable user_id here since the User model is now in the tipoff/authorization package which is required by this package.

Reply:
CC: @sl0wik Can you make this change and then I will merge?

Reply:
Done